inventory_management_system_api/repositories/usage_status.py

```diff
@@ -89,3 +110,13 @@ def _is_duplicate_usage_status(self, code: str, session: ClientSession = None) -
         """
         logger.info("Checking if usage status with code '%s' already exists", code)
         return self._usage_statuses_collection.find_one({"code": code}, session=session) is not None
+
+    def _is_usage_status_in_item(self, usage_status_id: str, session: ClientSession = None) -> bool:
+        """Checks to see if any of the documents in the database have a specific usage status id
```

Comment:
I think this should say
```suggestion
        """Checks to see if any of the items in the database have a specific usage status ID
```